### ISSUE TITLE
Support different CPU architectures

### ISSUE THREAD

**Issue by sephiroth-j** [webdriverextensions/webdriverextensions-maven-plugin#65]
As discussed in webdriverextensions/webdriverextensions-maven-plugin-repository#91
> The os.arch / cpu architecture would be good. It would be a new property of the driver object defaulting to "x86" if not set. CPU arch + platform + 64bit = better driver selection.

**Comment by nweiser94** [comment]
Hi @sephiroth-j , is there any progress on this topic ? I'd like to help whereever i can to speed up the integration of the Apple Silicon drivers into the webdriveextensions-maven-plugin-repository.

**Comment by sephiroth-j** [comment]
I just released a SNAPSHOT of version 3.4.0 which includes support for AARCH64. It uses [a special version of the repository file](https://github.com/webdriverextensions/webdriverextensions-maven-plugin-repository/blob/feature/support-for-cpu-arch/repository-3.0.json) which includes the following
- edgedriver 105.0.1343.53 for MAC and WINDOWS on ARM64
- chromedriver 105.0.5195.52 for MAC on ARM64
- geckodriver 0.31.0 for MAC on ARM64

It is also no problem to use this repo file with the current stable version 3.3.0 of the plugin. The new property is ignored and the plugin finds the 64-bit variant of the driver twice and selects the first one - i.e. the AMD64 variant.

Anyone who wants to test this version must change the version of the plugin in `pom.xml` to "3.4.0-SNAPSHOT" and add the Sonatype SNAPSHOT repository with url "https://oss.sonatype.org/content/repositories/snapshots/" as follows.

```xml
<repositories>
	<repository>
		<id>sonatype snapshots</id>
		<url>https://oss.sonatype.org/content/repositories/snapshots/</url>
	</repository>
</repositories>
```

**Comment by nweiser94** [comment]
Hi,
just tested this. Works fine for me.
